$OpenBSD$

EXPERIMENTAL: disable mono_get_lmf call that returns invalid
memory on OpenBSD

Index: mono/mini/mini-exceptions.c
--- mono/mini/mini-exceptions.c.orig
+++ mono/mini/mini-exceptions.c
@@ -3146,7 +3146,7 @@ mono_thread_state_init (MonoThreadUnwindState *ctx)
 #endif
 
 	ctx->unwind_data [MONO_UNWIND_DATA_DOMAIN] = mono_domain_get ();
-	ctx->unwind_data [MONO_UNWIND_DATA_LMF] = mono_get_lmf ();
+	ctx->unwind_data [MONO_UNWIND_DATA_LMF] = NULL;
 	ctx->unwind_data [MONO_UNWIND_DATA_JIT_TLS] = thread ? thread->jit_data : NULL;
 	ctx->valid = TRUE;
 }
